src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.php

```diff
@@ -116,5 +118,14 @@
                 service('property_info'),
             ])
             ->tag('validator.auto_mapper')
+
+        ->set('validator.password_strength_estimator.default', EntropyBasedPasswordStrengthEstimator::class)
+        ->alias('validator.password_strength_estimator', 'validator.password_strength_estimator.default')
+
+        ->set('validator.password_strength', PasswordStrengthValidator::class)
+            ->args([service('validator.password_strength_estimator')->nullOnInvalid()])
+            ->tag('validator.constraint_validator', [
+                'alias' => PasswordStrengthValidator::class,
```

Comment:
see https://github.com/symfony/symfony/pull/49850#discussion_r1151968345 about this line
I'm waiting for @stof's feedback on this ;)

Reply:
should be removed, as I did in #49850